
Header header
Gap[] gapArray

float32 max_depth
int32 max_depth_idx

float32 max_width
int32 max_width_idx



